theories/dune

```diff
@@ -0,0 +1,35 @@
+(coq.theory
+ (name Coq)
+ (public_name coq.Coq)
+ ; (package coq)
+ (synopsis "Coq's Standard Library")
+ (flags -q -allow-sprop)
```

Comment:
The `-allow-sprop` flag is not needed anymore.

Reply:
Good point, thanks.